### PR TITLE
Stringify objects passed to LogPayload

### LINKED ISSUE #6: Debugging an object doesn't show its properties
As mentioned by @slovenianGooner in https://github.com/m1guelpf/ray-js/issues/1#issuecomment-771035439, debugging an object doesn't show it properties.

image

### PR DESCRIPTION
**Pull request by karkowg**
Closes https://github.com/m1guelpf/ray-js/issues/6

### CODE REVIEW

**Comment by karkowg** [comment]
@m1guelpf if you agree with the implementation, could you verify that it works? Thanks.

File: src/utils/type.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Check type of operand with more specificity than `typeof`.
+ *   Slightly modified version of MDN helper found in `typeof` definition page.
+ * @link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#real-world_usage
+ *
+ * @param {*} obj
+ * @param {*} fullClass
+ * @returns {string}
+ */
+export default function type(obj, fullClass = false) {
```
Comment (m1guelpf) [inline src/utils/type.js]: Are we using the `fullClass` boolean? if not, remove it.

**Review by m1guelpf — CHANGES_REQUESTED** [review]

**Comment by cossou** [comment]
Hi @karkowg & @m1guelpf I think we should be using the `json_string` payload type.

Check the [spatie/ray/src/Payloads/JsonStringPayload.php](https://github.com/spatie/ray/blob/master/src/Payloads/JsonStringPayload.php)

**Comment by cossou** [comment]
What about:

`src/payloads/JsonPayload.js`
```js
export default (value) => {
  return {
      type: 'json_string',
      content: { value: JSON.stringify(value) } ,
  }
}
```

`src/index.js`
```js
    // ...

    send(...values) {
        if (values.length == 0) return this

        values.map(value => type(value) === 'object' ? this.json(value) : this.sendRequest(LogPayload(value)))

        return this
    }

    json(value) {
        this.sendRequest(JsonPayload(value))

        return this
    }

    // ...
```

**Comment by karkowg** [comment]
@cossou wouldn't your example send one request per payload instead of a single request with all payloads?

File: src/utils/type.js (hunk 1 above)
Comment (karkowg) [reply src/utils/type.js]: Removed.

**Comment by m1guelpf** [comment]
I do prefer the JSON payload approach, but sending multiple requests per payload is an issue, can we do something like this instead?

```js
    // ...

    send(...values) {
        if (values.length == 0) return this

        this.sendRequest(values.map(value => type(value) === 'object' ? JsonPayload(value) : LogPayload(value)))

        return this
    }

    json(value) {
        this.sendRequest(JsonPayload(value))

        return this
    }

    // ...
```

**Comment by karkowg** [comment]
Yep, I will update it asap.

**Review by m1guelpf — APPROVED** [review]
Thanks for your work on this, @karkowg!

**Pull request merged**